test/browser_tests/page_objects/page_the-bureau.js

```diff
@@ -6,6 +6,37 @@ function TheBureauPage() {
   };
 
   this.pageTitle = function() { return browser.getTitle(); };
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
+
+  this.hero = element( by.css( '.hero__bureau' ) );
+
+  this.heroVideoPlayer = this.hero.element( by.css( '.video-player') );
+
+  this.bureauHistory = element( by.css( '.bureau-history' ) );
+
+  this.bureauFunctions = element( by.css( '.bureau-functions' ) );
+
+  this.bureauMission = element.all(
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/8555b964c486de0b03084f0e58e3f97f44f7727a